app/models/school.rb

```diff
@@ -11,4 +11,8 @@ class School < ApplicationRecord
   validates :urn, presence: true, uniqueness: true
   validates :name, presence: true
   validates :lead_school, inclusion: { in: [true, false] }
+
+  def urn_and_location
```

Comment:
We have a very similar method to return the full name of a trainee, but it's in `TraineeHelper`. Should this be in a school helper as it's for view / presentational purposes?

Reply:
Yes, I think you're right. I'll create a helper method,